shapes/plot-description/plot-description-enhanced-protocol-shapes/slope/shapes.ttl

```diff
@@ -166,7 +166,7 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
         urnc:Requirement ;
     dcterms:source "Ecological Monitoring System - Australia Protocols" ;
     reg:status reg:statusSubmitted ;
-    sh:datatype xsd:float ;
+    sh:datatype xsd:integer ;
     sh:description "Value _MUST_ be between 0 and 90 inclusive." ;
     sh:maxInclusive 90 ;
```

Comment:
@javiersan18 - can you update this to 100; see this commit vocab_files/observable_properties_by_module/plot-description/plot-description-enhanced-protocol/slope.ttl